internal/components/exporters_test.go

```diff
@@ -195,7 +195,7 @@ func TestDefaultExporters(t *testing.T) {
 			assert.Equal(t, config.NewComponentID(tt.exporter), factory.CreateDefaultConfig().ID())
 
 			if tt.skipLifecycle {
-				t.Log("Skipping lifecycle test", tt.exporter)
+				t.Skip("Skipping lifecycle test", tt.exporter)
```

Comment:
Added this so that `go test` would should that the test was skipped instead of a log message.